src/libraries/Common/tests/LongName/LongName.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why does this project live in the Common folder? I would expect it under System.Diagnostics.Process\tests\LongPath\

Reply:
good point, moved